Ansible/ansible_collections/jfrog/platform/group_vars/all/vars.yml

```diff
@@ -0,0 +1,74 @@
+---
+# Defaults
+## Note : These values are global and can be overridden in role/<product>/defaults/main.yaml file
+## For production deployments,You may want to generate your master amd join keys and apply it to all the nodes.
+master_key: ee69d96880726d3abf6b42b97d2ae589111ea95c2a8bd5876ec5cd9e8ee34f86
```

Comment:
Why are these set to what appears to be valid master and join key. Should we instead have the customer generate these like we do for the helm chart?

Reply:
This for default installations, As you see a note in Readme for production deployments, this needs to be changes